src/js/controller/PaletteController.js

```diff
@@ -106,11 +110,21 @@
     } else {
       colorPicker.spectrum('set', color);
     }
-    this.setTitleOnPicker_(color, colorPicker);
   };
 
-  ns.PaletteController.prototype.setTitleOnPicker_ = function (title, colorPicker) {
-    var spectrumInputSelector = '.sp-replacer';
-    colorPicker.next(spectrumInputSelector).attr('title', title);
+  /**
+   * @private
+   */
+  ns.PaletteController.prototype.showValueDisplayer_ = function (evt) {
```

Comment:
Sorry to read that :/ Anything in particular you dislike about this ?

The tooltip supports HTML so we could improve its layout. Maybe using the hex string would be nicer ?